testsuite/features/step_definitions/cobbler_steps.rb

```diff
@@ -293,8 +293,8 @@
   else
     $server.run('systemctl restart cobblerd')
   end
-  # give cobbler a second to come up
-  sleep 1
+  # give cobbler a few seconds to come up
+  sleep 3
```

Comment:
As I mentioned in the previous PR from MC, I would prefer a smarter approach.
Having a constant value to sleep, we will depend on the environment and the workload we could have on the server.

It would be much better to use something like our step:
```
When(/^I wait until "([^"]*)" service is active on "([^"]*)"$/) do |service, host|
```

Reply:
this doesn't work unfortunately, I tried everywhere to add this step ```step %(I wait until "cobblerd" service is active on "server")``` but it didn't work, somehow systemd doesn't get the true status of cobbler.